ykman-gui/qml/ContentStack.qml

```diff
@@ -125,6 +125,14 @@ StackView {
         replaceAtDepth(3, pivChangeManagementKeyView, 'pivChangeManagementKeyView')
     }
 
+    function pivGetManagementKeyAndThen(pinCallback, keyCallback) {
```

Comment:
Could we rename it to `pivGetPinOrManagementKey` ?

Reply:
Sure!